src/App.test.jsx

```diff
@@ -20,17 +23,23 @@ const user = {
 }
 
 // 🚨 Create your server
+const server = setupServer(
+  rest.get('https://uzgiamkrbapxufnwdrja.supabase.co/rest/v1/users', (req, res, ctx) => {
+    return res(ctx.json([user]))
```

Comment:
We should validate the query params here. https://mswjs.io/docs/recipes/request-assertions